installers/Config.hs

```diff
@@ -68,32 +68,61 @@ argReadLower = arg (diagReadCaseInsensitive . unpack)
 data Backend
   = Cardano { cardanoDaedalusBridge :: FilePath }
   | Mantis
-  deriving (Eq, Read, Show)
+  deriving (Eq, Show)
+
+data Command
+  = GenConfig
+    { cfDhallRoot   :: Text
+    , cfOutdir      :: FilePath
+    }
+  | CheckConfigs
+    { cfDhallRoot   :: Text
+    }
+  | GenInstaller
+  deriving (Eq, Show)
 
 data Options = Options
   { oBackend        :: Backend
   , oBuildJob       :: Maybe BuildJob
+  , oOS             :: OS
   , oCluster        :: Cluster
   , oAppName        :: AppName
   , oDaedalusVer    :: Version
-  , oOutput         :: Text
+  , oOutput         :: FilePath
   , oPullReq        :: Maybe PullReq
   , oTestInstaller  :: TestInstaller
   , oCI             :: CI
   } deriving Show
 
-optionsParser :: Parser Options
-optionsParser = Options
+commandParser :: Parser Command
+commandParser = (fromMaybe GenInstaller <$>) . optional $
+  subcommandGroup "Subcommands:"
+  [ ("config",        "Build configs for an OS / cluster  (see: --os, --cluster top-level options)",
+      GenConfig
+      <$> argText "INDIR"  "Directory containing Dhall config files"
+      <*> (fromText <$> argText "OUTDIR" "Target directory for generated YAML config files"))
+  , ("check-configs", "Verify all Dhall-defined config components",
+      CheckConfigs
+      <$> argText "DIR" "Directory containing Dhall config files")
+  , ("installer",  "Build an installer",
+      pure GenInstaller)
+  ]
+
+optionsParser :: OS -> Parser Options
+optionsParser detectedOS = Options
   <$> backendOptionParser
   <*> (optional      $
       (BuildJob     <$> optText "build-job"           'b' "CI Build Job/ID"))
+  <*> (fromMaybe detectedOS <$> (optional $
+                   optReadLower "os"                  's' "OS, defaults to host OS.  One of:  linux macos64 win64"))
```

Comment:
I think this should be linux64

Reply:
Yes, I think so to. `linux64` is the de facto `SystemTag` we plan to use for Linux x86-64 builds.